### PR TITLE
Write documentation for the Timeline page.

### PR DESCRIPTION
**Pull request by kenzieschmoll**

### CODE REVIEW

File: docs/timeline.md [hunk 1]
```diff
@@ -2,4 +2,48 @@
...
+
+### Jank
+The frame rendering chart shows CPU and GPU jank in shades of red. We consider a portion of the frame to be janky when
+it takes more than 8 ms to complete.
+
+To achieve a frame rendering rate of 60 FPS (frames per second), each frame must render in ~16 ms or less. Since there
+are two portions of work for each frame (CPU and GPU), each portion should complete in 8 ms or less. When this target is
+missed, you may experience UI jank or dropped frames.
+
+## Frame Flame Chart
```
Comment (devoncarew) [inline docs/timeline.md]: `Frame Flame Chart` ==> `Flame Chart`?

File: docs/timeline.md [hunk 2]
```diff
@@ -2,4 +2,48 @@
...
+The flame chart shows the event trace for a single frame. The top-most event spawns the event below it, and so on and so
+forth. The CPU and GPU events are separate event flows, but they share a common timeline (displayed at the top of the
+flame chart). This timeline is strictly for the given frame. It does not reflect the clock shared by all frames.
+
+<img src="images/timeline_flame_chart.png" width="800" />
+
+The flame chart supports zooming and scrolling. You can click an event to view more details in the details section below
```
Comment (devoncarew) [inline docs/timeline.md]: You way additionally want to mention how to zoom and pan.

Comment (kenzieschmoll) [reply docs/timeline.md]: done.

File: docs/timeline.md (hunk 1 above)
Comment (kenzieschmoll) [reply docs/timeline.md]: done

**Pull request merged**